### DISCUSSION TITLE
How to use this plugin with TOML?

### DISCUSSION THREAD

**Discussion by paddyroddy** [tox-dev/tox-gh#141] [Q&A] [answered]
I'm migrating away from `legacy_tox_ini`. I currently have

```toml
[tool.tox]
env_list = [
    "py311",
    "py312",
]
env_run_base = {commands = [
    [
        "pytest",
        "--cov",
        "--cov-report=lcov",
    ],
], deps = [
    "pytest-cov",
]}
legacy_tox_ini = """
    [gh]
    python =
        3.11: py311
        3.12: py312
"""
```

but this actually doesn't work unless I comment out the `legacy_tox_ini` section. You probably can't mix the two, but I'm unsure how to use this plugin as the README only documents INI format.

**Comment by paddyroddy** [comment]
Is the dual macos ticks related to this?
![image](https://github.com/user-attachments/assets/3aaab72a-3aca-4c7e-9618-f5da001dac94)

**Reply by gaborbernat** [reply]
I don't think so, let me look at this.

**Reply by gaborbernat** [reply]
Can't replicate on this repo, so not sure what is up there.

**Reply by paddyroddy** [reply]
Hmm... I'll try the solution below and see if that solves the issue

**Reply by paddyroddy** [reply]
Yep that solved it!
<img width="316" alt="image" src="https://github.com/user-attachments/assets/4597d974-9eed-4cc4-89cc-30ecb268f248">

**Comment by gaborbernat** [comment, marked as answer]
Added now to https://github.com/tox-dev/tox-gh?tab=readme-ov-file#examples

**Reply by paddyroddy** [reply]
Great thanks!
